### PR TITLE
Propose update to simorgh code owners

### PR DESCRIPTION
**Pull request by chris-hinds**
Proposing an update to our current code owners strategy. Keep existing code owners on important files, pipelines, env config, pipeline scripts etc.

We should still look to expand the code owners of these files in the future too.

### CODE REVIEW

**Review by karinathomasbbc — APPROVED** [review]
👍

**Review by craigkj — APPROVED** [review]
Im happy with this.

The lists of names attached to the remaining three pieces (/*webpack*, /envConfig/, Jenkinsfile) - is seems to be the admins plus a couple. Is there a name for this group that could be made into a team? Is there a reason why that isn't just the `@bbc/simorgh-admins` too?

Not blocking on this question, just curious.

**Comment by chris-hinds** [comment]
> Im happy with this.
> 
> The lists of names attached to the remaining three pieces (/_webpack_, /envConfig/, Jenkinsfile) - is seems to be the admins plus a couple. Is there a name for this group that could be made into a team? Is there a reason why that isn't just the `@bbc/simorgh-admins` too?
> 
> Not blocking on this question, just curious.

Because Simorgh Admins is just 3 people. Myself, James Donohue and Benjamin Hobbs.

At the time the code owners couldn't be a team as we had/have different code owners for different files/repos

**Review by jamesdonoh — APPROVED** [review]

**Pull request merged**
